webpack.config.dev.js

```diff
@@ -11,6 +11,7 @@ module.exports = merge(baseConfig, devConfig, {
     alias: {
       react: path.resolve('./node_modules/react'),
       'react-router-dom': path.resolve('./node_modules/react-router-dom'),
+      '@material-ui/core': path.resolve('./node_modules/@material-ui/core'),
```

Comment:
why do we need this ?

Reply:
When using npm link with centreon-ui, we get two instances of material-ui which leads to errors (same for react-router)